.github/workflows/release_build.yml

```diff
@@ -164,3 +231,36 @@ jobs:
         release_name: Release ${{ github.ref }}
         draft: false
         prerelease: false
+
+    - name: Upload Documentation To Release
+      if: steps.create_release.conclusion == 'success'
+      id: upload-release-asset
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: fusion-engine-client-docs.tar.gz
+        asset_name: fusion-engine-client-docs.tar.gz
+        asset_content_type: application/tar+gzip
+
+    - name: Upload Documentation To Webpage
```

Comment:
i guess lets pull this out in favor of s3, if you want to leave it for now that's fine too.